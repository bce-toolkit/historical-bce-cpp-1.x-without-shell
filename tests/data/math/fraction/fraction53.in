84353297600632682761234585886392194446577685791133773
77399432208877302199343425777458796501323944521173673
21216171220021332865716416987015949995018914395619865
25493577243892367570335103030808676578339091874677424
