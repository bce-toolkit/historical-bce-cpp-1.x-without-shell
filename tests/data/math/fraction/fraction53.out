3792586914234744810000270496785898498819631509038790239912478236128568519963402643063552326475973760854897/1973188403650424357193967763849732390182115440944718492414463706415817258030661592015200679250344756258352
508347702082795211746687312415619715798602912885157070410327820942601059332059834100739593305293453226607/1973188403650424357193967763849732390182115440944718492414463706415817258030661592015200679250344756258352
1789654004868437675360746608999330540658260252147696551088929938706637679493229980081548403883283471200645/1973188403650424357193967763849732390182115440944718492414463706415817258030661592015200679250344756258352
2150467308158770010873478904600759107309117210961973655161403028535584789647731238582145959890633607040752/1642119606075974799126791592185139391510514298076816584751075207592983730315671404481406366585340153814145
0
1
0
0
1
1
